contracts/seed/Seed.sol

```diff
@@ -159,6 +159,9 @@ contract Seed {
     */
     function buy(uint256 _amount) public protected checked {
         require((fundingToken.balanceOf(address(this)).add(_amount)) <= cap, "Seed: amount exceeds contract sale cap");
+        require( seedToken.balanceOf(address(this)) >=
+        (fundingToken.balanceOf(address(this)).add(_amount)).mul(price).div(PCT_BASE),
+            "Seed: seed distribution exceeded");
         require(fundingToken.transferFrom(msg.sender, address(this), _amount), "Seed: no tokens");
```

Comment:
pls transfer `_amount.mul(price)`

Reply:
In `172: _addLock(msg.sender, (_lockTokens.add((_amount).mul(price).div(PCT_BASE))), _amount, feeAmount);` pls adjust `seedAmount` and `fundingAmount`

Reply:
Done